packages/components/Notification/src/NotificationTokens.ts

```diff
@@ -1,6 +1,7 @@
 import { Theme } from '@fluentui-react-native/framework';
 import { TokenSettings } from '@fluentui-react-native/use-styling';
 import { NotificationTokens } from './Notification.types';
+import { notificationColors } from './NotificationTokens.ios';
```

Comment:
Yeah this will fail because metro won't include the base file `notificationTokens.ts` in your bundle, I'm pretty sure.